tests/wpunit/PatchSettingsTest.php

```diff
@@ -0,0 +1,401 @@
+        parent::tearDown();
+    }
+
+    // tests
+    public function testPatchSettingsQuery()
+    {
+        $mock_options = [
+			'default_pingback_flag' => 1,
+            'default_comment_status' => 'open',
+            'comments_notify' => 1,
+            'moderation_notify' => 1,
+            'comment_moderation' => true,
+            'require_name_email' => 1,
+            'comment_whitelist' => 1,
+            'comment_max_links' => 2,
+            'moderation_keys' => 'Yep\nKey\nList',
+            'blacklist_keys' => 'Yep\nKey\nList',
+            'show_avatars' => 1,
+            'avatar_rating' => 'G',
+            'avatar_default' => 'mystery',
+            'close_comments_for_old_posts' => true,
+            'close_comments_days_old' => 14,
+            'thread_comments' => 1,
+            'thread_comments_depth' => 5,
+            'page_comments' => true,
+            'default_comments_page' => 'newest',
+            'comment_order' => 'asc',
+            'comment_registration' => true,
+            'show_comments_cookies_opt_in' => false,
+            'thumbnail_size_w' => 150,
+            'thumbnail_size_h' => 150,
+            'thumbnail_crop' => 1,
+            'medium_size_w' => 300,
+            'medium_size_h' => 300,
+            'large_size_w' => 1024,
+            'large_size_h' => 1024,
+            'posts_per_rss' => 10,
+            'rss_use_excerpt' => true,
+            'show_on_front' => 'posts',
+            'page_on_front' => '1',
+            'page_for_posts' => '2',
+            'blog_public' => true,
+            'default_email_category' => 1,
+            'default_link_category' => 2,
+            'permalink_structure' => '/archives/%post_id%',
+            'category_base' => 'kate',
+            'tag_base' => 'boggy',
+            'wp_page_for_privacy_policy' => '2'
+		];
+
+		foreach ( $mock_options as $mock_option_key => $mock_value ) {
+			update_option( $mock_option_key, $mock_value );
+        }
+        
+        $query  = "
+			query {
+                discussionSettings{
+                    defaultPingbackFlag
+                    defaultCommentStatus
+                    commentsNotify
+                    moderationNotify
+                    commentModeration
+                    requireNameEmail
+                    commentWhitelist
+                    commentMaxLinks
+                    moderationKeys
+                    blacklistKeys
+                    showAvatars
+                    avatarRating
+                    avatarDefault
+                    closeCommentsForOldPosts
+                    closeCommentsDaysOld
+                    threadComments
+                    threadCommentsDepth
+                    pageComments
+                    defaultCommentsPage
+                    commentOrder
+                    commentRegistration
+                    showCommentsCookiesOptIn
+                }
+                mediaSettings {
+                    thumbnailSizeW
```

Comment:
not _that_ big of a deal, but it kinda bugs me that this is named `thumbailSizeW` instead of `thumbnailSizeWidth` 🤔 

how are other image sizes handled, like when you `add_image_size()` (https://developer.wordpress.org/reference/functions/add_image_size/)

Reply:
wait, actually, there aren't `settings` for additional sizes, because you register them statically. . .duh. . .lol

disregard. 🤦‍♂️